src/components/Condition/UserRoleCondition.tsx

```diff
@@ -0,0 +1,20 @@
+import { PropsWithChildren } from 'react'
+import { AdminLayout } from '@layouts/AdminLayout'
+import { KaryawanLayout } from '@layouts/KaryawanLayout'
+import { useMarContext } from 'src/context/MarFashionProvider'
+
+export const UserRoleCondition = ({ children }: PropsWithChildren) => {
```

Comment:
better not put this on components. its depends on context. i think this is considered as layout. rename `UserRoleCondition` also maybe like `UseLayout`

Reply:
done. rename UserRoleCondition to UserLayout and move it from components to layouts